__std__/nix/automation/hydraJobs/all.nix

```diff
@@ -0,0 +1,63 @@
+{ inputs, cell }:
+
+let
+  inherit (inputs.cells.toolchain.library) pkgs haskell-nix;
+  inherit (pkgs.stdenv) system;
+  inherit (pkgs) lib;
+
+  make-haskell-jobs = project:
+    let
+      packages = haskell-nix.haskellLib.selectProjectPackages project.hsPkgs;
+    in
+    {
+      exes = haskell-nix.haskellLib.collectComponents' "exes" packages;
+      tests = haskell-nix.haskellLib.collectComponents' "tests" packages;
+      benchmarks = haskell-nix.haskellLib.collectComponents' "benchmarks" packages;
+      libraries = haskell-nix.haskellLib.collectComponents' "library" packages;
+      checks = haskell-nix.haskellLib.collectChecks' packages;
+      roots = project.roots;
+      plan-nix = project.plan-nix;
+    };
+
+  plutus-project = inputs.cells.plutus.library.plutus-project;
+
+  native-plutus-jobs = make-haskell-jobs plutus-project;
+
+  windows-plutus-jobs = make-haskell-jobs plutus-project.projectCross.mingwW64;
+
+  other-jobs = {
```

Comment:
Looking at this makes me think: what if we defined `ciJobs` per cell and then just harvested them all? That's kind of what's going on here: each cell is defining some jobs. Then `automation` would go away as a cell. Not sure if you find that better or worse organized tbh!

Reply:
I guess you would still need to define the top-level jobset and the required job :thinking: